src/sciline/_provider.py

```diff
@@ -23,7 +24,12 @@
 
 
 ToProvider = Callable[..., Any]
-ProviderKind = Literal['function', 'parameter', 'series', 'table', 'sentinel']
+"""Callable that can be converted to a provider."""
+
+ProviderKind = Literal[
+    'function', 'parameter', 'series', 'table', 'sentinel', 'unsatisfied'
```

Comment:
Is 'sentinel' still in use?

Reply:
Yes, in https://github.com/scipp/sciline/blob/011f9bfb6722c8348fa98f3d2b08ec9844909372/src/sciline/pipeline.py#L587 and https://github.com/scipp/sciline/blob/011f9bfb6722c8348fa98f3d2b08ec9844909372/src/sciline/pipeline.py#L591

Reply:
Ok, but it will never make it to, e.g., `visualize`, right?

Reply:
It shouldn't.